app/src/test/java/org/oppia/android/app/activity/route/ActivityRouterModuleTest.kt

```diff
@@ -102,17 +109,36 @@ class ActivityRouterModuleTest {
   }
 
   @Test
-  fun testInjectActivityRouterModule_isInstanceOfActivityRouter() {
-    val activityRouter = retrieveActivityRouter()
-    assertThat(activityRouter).isInstanceOf(ActivityRouter::class.java)
+  fun testInjectActivityRouterModule_routeToRecentlyPlayedActivity() {
+    activityRule.scenario.onActivity { activity ->
```

Comment:
An activity isn't needed. You can inject a context and pass that in instead of using an activity, e.g.:

```kotlin
@Inject lateinit var context: Context
```